include/swift/Driver/Compilation.h

```diff
@@ -70,6 +70,10 @@ class Compilation {
   /// The DiagnosticEngine to which this Compilation should emit diagnostics.
   DiagnosticEngine &Diags;
 
+  /// The ToolChain this Compilation was built with, that it may reuse to build
+  /// subsequent BatchJobs.
+  const ToolChain &TheToolChain;
```

Comment:
Okay, now we're getting an unused variable warning for this. Can you add `LLVM_ATTRIBUTE_UNUSED` to this until the rest of it lands?